src/tests/test_commander.py

```diff
@@ -1,13 +1,13 @@
-import pytest 
-import unittest
+# import pytest 
```

Comment:
pretty sure these are required for pytest to pick them up. What happened?

Reply:
These were unused by the code